### ISSUE TITLE
How to use tags?

### ISSUE THREAD

**Issue by rohrkemper** [jupyterlab/jupyterlab-toc#87]

**Comment by tomzx** [comment]
You need to install https://github.com/jupyterlab/jupyterlab-celltags and use it to add tags to your cells. Then you can filter by tag in the table of content extension.

**Comment by TristanK27** [comment]
Since jupyterlab:2.0.0 is part of jupyterlab, You can access it by clicking the 'property inspector' in the toolbar on the left'.

**Comment by Biaggio74** [comment]
@TristanK27 do you mean to edit the Metadata for a certain cell? 
```json
{
    "tags": [
        "parameters"
    ]
}
```
Do you know other ways how to create tags? Like in Jupyter Notebook, with a certain GUI style, just add a tag for each cells we wanted?

<img width="876" alt="Screenshot 2020-05-05 at 08 16 24" src="https://user-images.githubusercontent.com/11397475/81039240-c04ad100-8ea8-11ea-9b06-547f75fe6940.png">

**Comment by TristanK27** [comment]
@Biaggio74 Yes there is a GUI option to do so, here you can define your own tags with the 'add tag' button and toggle them on/off for the cells where you need them:
![afbeelding](https://user-images.githubusercontent.com/20356336/81561877-31e8ba80-9394-11ea-933c-48369c69016a.png)

**Comment by Biaggio74** [comment]
@TristanK27 my apologies for the late reply, ...how do you bring up this GUI you mentioned?

**Comment by MaxGhenis** [comment]
The GUI to add cell tags is in `View...Show right sidebar`

**Comment by graphedge** [comment]
All, the tag hide-output is not working for me in Lab 2.2.6 .  

Windows 10 + Firefox 90.0.2 64-biy

![image](https://user-images.githubusercontent.com/45630928/128414823-4c5a0338-76ae-4af6-8a68-de8728896b19.png)

Here's the notebook metadata.

{
    "kernelspec": {
        "display_name": "Python 3",
        "language": "python",
        "name": "python3"
    },
    "language_info": {
        "codemirror_mode": {
            "name": "ipython",
            "version": 3
        },
        "file_extension": ".py",
        "mimetype": "text/x-python",
        "name": "python",
        "nbconvert_exporter": "python",
        "pygments_lexer": "ipython3",
        "version": "3.8.5"
    }
}

**Comment by fcollonval** [comment]
@graphedge You can collapse the output and save the notebook. The _collapse_ state will be saved (using cell metadata not tags): https://youtu.be/WgiX3ZRaTiY

**Comment by graphedge** [comment]
@fcollonval True, but the error comes back when the student accidentally runs the cell, and that doesn't help them notice actual Python (in our case) errors.

Any idea what is preventing the hide-output tag from working ?

**Comment by fcollonval** [comment]
As far as I remember the trouble is linked to the the tag being non standard. And I don't recall a work-around.

> In your specific case, you could use python to output the data in a nice format instead of using non-valid cell. Something like:

```py
import json
json.dumps({"codeCellConfig": {"lineNumbers": True, "codeFolding": True}}, indent=4)
```

**Comment by graphedge** [comment]
Here's another solution.  Use a markdown cell with backticks to embed "code style" formatting.
[markdown notes](https://www.freecodecamp.org/news/how-to-format-code-in-markdown/)
